syntax = "proto3";
package syft.core.ast;

message AstModule {
  string obj_type = 1;
  bytes data = 2;
}
